packages/remix-dev/compiler/compilerServer.ts

```diff
@@ -63,6 +63,17 @@ const createEsbuildConfig = (
     plugins.unshift(NodeModulesPolyfillPlugin());
   }
 
+  if (config.serverBuildTarget === "netlify-edge") {
```

Comment:
We had this previously in the experimental edge branch.

Reply:
This the only extra adapter-specific check we do, so I'm wondering if this is still necessary or not?